docs/new-architecture-app-modules-android.md

```diff
@@ -430,8 +594,25 @@ public class MyApplication extends Application implements ReactApplication {
         ReactFeatureFlags.useTurboModules = true;
         //...
     }
+}
+```
+
+</TabItem>
+
+<TabItem value="kotlin">
+
+```kotlin
+class MyApplication : Application(), ReactApplication {
+    fun onCreate() {
```

Comment:
```suggestion
    override fun onCreate() {
```

Reply:
added override key to function declaration